src/backend/alembic/versions/2024_07_16_ed17f144f4bf_add_default_user.py

```diff
@@ -11,7 +11,8 @@
 import sqlalchemy as sa
 from alembic import op
 
-from src.backend.schemas.user import DEFAULT_USER_ID, DEFAULT_USER_NAME
+DEFAULT_USER_ID = "user-id"
```

Comment:
Just out of curiosity, why are we redefining these variables here?

Reply:
migrations should never import from the main code, since the main code can change, where-as migrations are like constants.